EIPS/eip-acceptance.md

```diff
@@ -0,0 +1,29 @@
+## Preamble
+
+    EIP: <to be assigned>
+    Title: EIP Community Veto
+    Author: Sam Griffin (sam.griffin@gmail.com)
+    Type: Meta
+    Status: Draft
+    Created: 2018-02-24
+    Modifies: EIP-1
+
+## Simple Summary
+The greater Ethereum community should be able to veto EIPs they don't like. Also, the process by which EIPs become part of the Ethereum standard should be clarified. 
+
+## Abstract
+Some EIPs have contentious moral/ethical/legal/philosophical rammifications. The process of drafting the EIP where the author comes up with what they think is the best form of their idea should be free from interference, but the community should have a voice to prevent unpopular EIPs becoming part of the standard. 
+
+## Motivation
+The community currently has no official procedure for indicating their dislike of certain EIPs. This leads to brigading of GitHub pull requests that make it more difficult for EIP editors and draft writers. Additionally, EIP draft writers do not have specific data on how controversial their EIP is. Controversial hard forks should be avoided whenever possible, and if compromise can happen in the EIP drafting stage, this could avoid community splits. The current role of the AllCoreDevs group is also not detailed in EIP-1, and this should be made explicit.
```

Comment:
> The community currently has no official procedure for indicating their dislike of certain EIPs.

This isn't true.  They indicate their dislike by not upgrading their nodes to run protocol changes.  The community's role in the governance process (currently) is that of final veto power.  While I appreciate that many people don't like this mechanism and prefer majority (mob) rule, it is incorrect to state that the community has no official procedure.

Reply:
> Controversial hard forks should be avoided whenever possible

You should specify in the EIP _why_ you believe that controversial hard forks should be avoided.

I _personally_ think that controversial hard forks are the single advantage that this type of governance has over traditional democratic governance.

Reply:
> The current role of the AllCoreDevs group is also not detailed in EIP-1

Agreed, and EIP-1 should be updated to reflect that.  However, creating an EIP that suggests that someone update another EIP for clarity isn't the proper procedure, instead I recommend just submitting a PR for EIP that adds the clarity.

Reply:
A problem with hard forks is that for every fork, people would have to dedicate time and effort to build clients on the new fork and implement changes (e.g. ones that they agree on from EIPs). Voting can provide signalling for decision making on whether to proceed with a change on Ethereum if there is a majority vote (which could also include criteria for key stakeholders like clients, miners/validators, core devs, etc.), otherwise, if not, people can choose to fork Ethereum and maintain the fork as they see fit.

Reply:
Yes, the people dedicating resources to authoring a fork are making a bet on where they believe people will trade.  However, I'm not convinced that a vote would actually be a valid signal as to the correctness of their actions.  As an example, I would have voted _against_ TheDAO, but ultimately when push came to shove I stayed with ETH instead of ETC.  If we assume (just for the sake of this discussion) that everyone was in the same boat then the vote would have changed the behavior of the client developers in a way that may not actually have been in the best long-term interest of Ethereum.

At the time of The DAO, I wasn't as well informed as I am now and if the vote were to happen today I _may_ vote differently (I'm not actually sure which way I would vote).  The problem is that the vast majority of people (like me) who would be participating in this vote are not as well informed as the people coding the clients.  It is just the nature of how the world works, often many people who are affected by a particular issue have not devoted a massive amount of time into researching and understanding that issue enough to make the most wise decision.

Reply:
I think it's great that cryptocurrency has the option for communities to peaceably split via hard forks, but I also think it should be the absolute last resort. If a compromise can be found before it reaches the point of a community split, why not?

@MicahZoltu surely you don't think that community splits should be common occurrences, do you?